lib/ansible/module_utils/facts.py

```diff
@@ -99,6 +99,7 @@ class Facts(object):
                     ('/etc/os-release', 'SuSE'),
                     ('/etc/gentoo-release', 'Gentoo'),
                     ('/etc/os-release', 'Debian'),
+                    ('/etc/debian_version', 'Debian'),
```

Comment:
Yes, I had to add this as /etc/os-release does not exist on Squeeze, only Wheezy and higher.
So adding this path allows the get_distribution_facts() function to work properly, since it loops based on:

```for (path, name) in Facts.OSDIST_LIST:```